requirements.sh

```diff
@@ -17,9 +17,25 @@
 ##--------------------------------------------------------------------
 
 ##
-## Author: Ashish Jabble
+## Author: Ashish Jabble, Massimiliano Pinto
 ##
 
 set -e
 
-sudo apt -y install libcurl4-openssl-dev
+set -e
+
+foglamp_location=`pwd`
+os_name=`(grep -o '^NAME=.*' /etc/os-release | cut -f2 -d\" | sed 's/"//g')`
+os_version=`(grep -o '^VERSION_ID=.*' /etc/os-release | cut -f2 -d\" | sed 's/"//g')`
+echo "Platform is ${os_name}, Version: ${os_version}"
+
+if [[ ( $os_name == *"Red Hat"* || $os_name == *"CentOS"* ) &&  $os_version == *"7"* ]]; then
+	sudo yum -y install curl
+	sudo yum -y install libcurl
+	sudo yum -y install curl-devel
+	sudo yum -y install libcurl-devel
```

Comment:
remove `sudo` commands, the script needs to be executed with sudo

Reply:
If we remove sudo, and launch this script via sudo, then the permissions of created files are all set to root.

The only action needed by root is related to yum commands